registries/adapters/openshift_adapter_test.go

```diff
@@ -89,11 +89,6 @@ const AuthResponse = `
   "issued_at": "2018-03-27T19:54:19Z"
 }`
 
-func TestOpenShiftName(t *testing.T) {
-	ocpa := OpenShiftAdapter{}
-	ft.Equal(t, ocpa.RegistryName(), "openshift", "openshift name does not match `openshift`")
```

Comment:
I would prefer we don't delete the test, rather check that the prefix was removed.